app/javascript/components/mentoring/Inbox.jsx

```diff
@@ -9,13 +9,16 @@ import { usePaginatedRequestQuery } from '../../hooks/request-query'
 import { useIsMounted } from 'use-is-mounted'
 import { ResultsZone } from '../ResultsZone'
 import { useHistory } from '../../hooks/use-history'
-import { useDebounce } from '../../hooks/use-debounce'
 
 export function Inbox({ tracksRequest, sortOptions, ...props }) {
-  const { request, setCriteria, setOrder, setPage, setQuery } = useList(
-    props.discussionsRequest
-  )
-  const debouncedQuery = useDebounce(request.query, 500)
```

Comment:
What's the difference between the old code and the new code?

Reply:
Previously everything was being routed via the debounce (including changing pagination-pages, changing tabs), etc. Now only the textual search goes via it.

Reply:
Aha, okay!